frontend/Pages/PosterScreen.razor

```diff
@@ -3,82 +3,113 @@
 @using System.Collections.Generic
 @using frontend.Shared.Layout
 @layout PosterScreenLayout
+@inject IJSRuntime JS
+
+
+<p> <button @onclick="ShowSlides">Show slides!</button> </p>
 
 <div class="container">
-    <div class="loading" hidden=@hidetext>
+
+    <!-- <div class="loading" hidden=@hidetext>
         <h1> Loading picture number: @currentPicture ...</h1>
-    </div>
-    <div class="imgContainer" hidden=@hidepicture>
-        <img class="poster" src="@imageSource">
+    </div> -->
+
+    <div class="imgContainer" <img class="poster" src="@currentPicture">

```

Comment:
This should be `<div class="imgContainer"><img class="poster" src="@currentPicture"` /></div>